nixos/modules/config/users-groups.nix

```diff
@@ -69,6 +69,10 @@ let
     {option}`users.mutableUsers` is false, you cannot change
     user passwords, they will always be set according to the password
     options.
+
+    NOTE: one exception is if `systemd.sysusers.enable = true;`. In this case
```

Comment:
I think you can cross-reference this option with `[](#opt-systemd.sysusers.enable)`.